### ISSUE TITLE
Feature request: dataless2stationxml script

### ISSUE THREAD

**Issue by claudiodsf** [obspy/obspy#1878]
Hello, now that #1185 is merged, wouldn't it make sense to have a `obspy-dataless2stationxml` script?

**Comment by barsch** [comment]
we already have some conversion scripts - but I consider that rather bad - it would be better to have a more general converter script, or? also for all the event/waveform formats out there ...

**Comment by claudiodsf** [comment]
Yes, in principle we would need three generic scripts: `waveform_converter`, `metadata_converter`, `event_converter`, or something like this...

**Comment by megies** [comment]
Who needs converters, when you can do all processing inside obspy..?  😜 

Just kidding.. I agree with @barsch that this should be handled by a single CLI script probably, current scripts maybe should be deprecated in the process..

```
$ obspy-convert -i my_dataless.seed -o my_station.xml -f STATIONXML
```

Even specifying what type of data should be output (waveforms, stations, events) can be omitted and autodetected for almost all output formats specs. Only very few can represent more than one data type I think (only KML, GSE2, SHAPEFILE is ambiguous I think?)
